cache log - delete image failed

### Describe your problem!

See CI Watchdog test log, e.g. https://ci.cgeo.org/job/WatchdogTests_master/5038/
~~~
cgeo.watchdog.WatchdogTest > testGeocachingLogCache[android-30(AVD) - 11]  [31mFAILED  [0m
	java.lang.AssertionError: FAILED for 'deleteImage'(logId = GL1GP4D2G): java.lang.RuntimeException: java.net.SocketTimeoutException: timeout
	at cgeo.geocaching.connector.gc.GCLogAPITest.executeTest(GCLogAPITest.java:233)
~~~

Maybe a change at the GC connector?
Used path:
`/api/live/v1/images/delete/GL1GP7GRA/f55b5aae-c27c-4c27-ba3c-ed6088faabf6`


### How to reproduce?

see above

### Actual result after these steps?

see above

### Expected result after these steps?

success

### Reproducible

Yes

### c:geo Version

Nightly 20260606

### System information

```text

```

### Additional Information

_No response_

--- a/main/src/main/java/cgeo/geocaching/connector/gc/GCLogAPI.java
+++ b/main/src/main/java/cgeo/geocaching/connector/gc/GCLogAPI.java
@@ -368,8 +368,9 @@ public class GCLogAPI {
         }
 
         //2,) Send a DELETE Request (which is a POST)
-        try (HttpResponse response = websiteReq().uri("/api/live/v1/images/delete/" + logId + "/" + getGuidFrom(logImageId))
+        try (HttpResponse response = websiteReq().uri("/api/live/v1/trpc/web.images.delete?batch=1")
             .method(HttpRequest.Method.POST)
+            .body("{\"0\":{\"refCode\":\"" + logId + "\",\"guid\":\"" + getGuidFrom(logImageId) + "\"}}")
             .headers(HTML_HEADER_CSRF_TOKEN, csrfToken)
             .request().blockingGet()) {
 

